catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/SpriteListFragment.kt

```diff
@@ -286,7 +286,7 @@ class SpriteListFragment : RecyclerViewFragment<Sprite?>() {
         }
     }
 
-    fun addFromLibrary(selectedItem: Sprite?) {
+    private fun addFromLibrary(selectedItem: Sprite?) {
```

Comment:
This was a harmless change but I'm not sure if it should be a part of this PR?

Reply:
Change has no effect on the behavior of the application and improves code quality. I see no problem to approve this change.